images/manageiq-base/Dockerfile

```diff
@@ -48,33 +48,42 @@ LABEL name="manageiq-base" \
       io.k8s.description="ManageIQ is a management and automation platform for virtual, private, and hybrid cloud infrastructures." \
       io.openshift.tags="ManageIQ,miq,manageiq"
 
-RUN yum -y install --setopt=tsflags=nodocs \
-                   chrony           \
-                   cmake            \
-                   cronie           \
-                   file             \
-                   gcc-c++          \
-                   git              \
-                   http-parser      \
-                   initscripts      \
-                   libcurl-devel    \
-                   libpq            \
-                   libtool          \
-                   libxslt-devel    \
-                   logrotate        \
-                   lvm2             \
-                   net-tools        \
-                   nmap-ncat        \
-                   openldap-clients \
-                   openscap-scanner \
-                   patch            \
-                   psmisc           \
-                   postgresql-devel \
-                   sqlite-devel     \
-                   sysvinit-tools   \
-                   which            \
-                   &&               \
-    yum clean all
+RUN dnf -y --disableplugin=subscription-manager install http://mirror.centos.org/centos/8/BaseOS/${ARCH}/os/Packages/centos-repos-8.1-1.1911.0.8.el8.${ARCH}.rpm http://mirror.centos.org/centos/8/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8.1-1.1911.0.8.el8.noarch.rpm && \
```

Comment:
A note... because centos-repos is being added here, most of the packages installed below is coming from CentOS repos, not UBI8 repos, even if available in UBI8. It's the same version, but different binaries built by CentOS. If a newer version is available in UBI8 repo, that will be picked up though.